src/signals/incident/components/form/MapSelectors/Asset/Selector/DetailPanel/DetailPanel.test.tsx

```diff
@@ -22,7 +22,7 @@ jest.mock('react-responsive')
 
 jest.mock('../../AssetList', () =>
   // eslint-disable-next-line @typescript-eslint/no-unused-vars
```

Comment:
This linter disable comment shouldn't be necessary any more

Reply:
ok